config/checkstyle_checks.xml

```diff
@@ -371,9 +371,10 @@
             DetailsAST, CheckstyleException, UnsupportedEncodingException, BuildException, ConversionException, FileNotFoundException, TestException"/>
     </module>
     <module name="ClassFanOutComplexity">
-        <property name="max" value="25"/>
+        <!-- 25 => 26 after added support extends, implements and annotation class Checker.java failed-->
+        <property name="max" value="26"/>
```

Comment:
Do not change checkstyle config because of one file, please use suppressions.xml. Also when you add smth in suppressions you have to explain in comment why it cannot/should not be fixed.

Reply:
fixed